include/cutlass/fast_math.h

```diff
@@ -1,5 +1,6 @@
 /***************************************************************************************************
  * Copyright (c) 2017 - 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ * Copyright (c) 2024 - 2024 Codeplay Software Ltd. All rights reserved.
```

Comment:
We usually don't add the Codeplay copyright unless we have to rewrite a significant part of the file.

Reply:
removed